playbooks/setup-newmachine.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Is this intentional?

Reply:
seems to be, you have it in many places

Reply:
Yeah. The linter did it. I think it's standard yaml.